go/interpreter/lfvm/opcode.go

```diff
@@ -221,7 +221,7 @@ const (
 	// Not really an Op-code but used to get the number of executable opcodes.
 	NUM_EXECUTABLE_OPCODES
 
-	// Special no-instrictions op codes
+	// Special no-instructions op codes
```

Comment:
since you are changing it, what about 
```suggestion
	// Special non-instruction op codes
```